api/prisma/migrations/26_update_combined_listing/migration.sql

```diff
@@ -0,0 +1,222 @@
+DROP VIEW IF EXISTS "combined_listings";
+
+CREATE VIEW "combined_listings" AS (
```

Comment:
There are two changes to this view. 
1. Listings are only added to this view if they have status 'active'. This will shrink the view in production from 716 listings to 70 which should significantly improve performance
2. The units are no longer in this view, but rather a new view. When there are a lot of units for a listing the postgres queries really struggle. Even if we aren't querying on that field. This aspect has the biggest performance improvement of anything done in this PR